tensorflow/core/kernels/conv_ops_test.cc

```diff
@@ -770,7 +770,15 @@ class FusedConv2DOpTest : public OpsTestBase {
     ASSERT_EQ(conv_2d.dtype(), fused_conv_2d.dtype());
     ASSERT_EQ(conv_2d.shape(), fused_conv_2d.shape());
 
-    test::ExpectClose(conv_2d, fused_conv_2d, /*atol=*/1e-6);
+    // NOTE(intel-tf): When filter_size is equal to the input image size,
+    // conv2d essentially is element-wise multiplication followed by
+    // a full sum reduction, which causes larger numerical error
+    // than usual cases.
+    if (image_width == filter_size && image_height == filter_size) {
+      test::ExpectTensorNear<T>(conv_2d, fused_conv_2d, 1e-4);
```

Comment:
Could you please try if 
```suggestion
      test::ExpectClose(conv_2d, fused_conv_2d, 1e-4);
```
works? Or `test::ExpectClose(conv_2d, fused_conv_2d, 1e-4, 1e-4);`?
`ExpectTensorNear` is absolute value comparison and I prefer to avoid it.

Reply:
@penpornk (1) is actually not numerical precision rather no mkl kernel found error for `double`. Actually, we only have `float` version of the fusion. Our code has been updated here (`tensorflow/core/graph/mkl_layout_pass.cc`).

Yes, ExpectClose has also work for us. Will make changes there.